lib/go-atscfg/headerrewritedotconfig_test.go

```diff
@@ -496,42 +407,91 @@ func TestGetAssignedMids(t *testing.T) {
 			ID:         util.Ptr(2),
 			Status:     util.Ptr(string(tc.CacheStatusReported)),
 		},
-
+	}
+	mids := []Server{
 		{
 			Cachegroup: util.Ptr("mid1"),
 			CDNName:    util.Ptr("mycdn"),
 			HostName:   util.Ptr("midCache1"),
 			ID:         util.Ptr(3),
 			Status:     util.Ptr(string(tc.CacheStatusReported)),
+			Type:       tc.MidTypePrefix,
 		},
 		{
 			Cachegroup: util.Ptr("mid2"),
 			CDNName:    util.Ptr("mycdn"),
 			HostName:   util.Ptr("midCache2"),
 			ID:         util.Ptr(4),
 			Status:     util.Ptr(string(tc.CacheStatusReported)),
+			Type:       tc.MidTypePrefix,
 		},
 	}
+	allServers := append(edges, mids...)
+	_ = allServers
 
-	allDeliveryServices := []DeliveryService{{}, {}}
+	topology := tc.Topology{
+		Name: "mytopology",
+		Nodes: []tc.TopologyNode{
+			{
+				Cachegroup: "edge1",
+				Parents:    []int{2},
+			},
+			{
+				Cachegroup: "edge2",
+				Parents:    []int{2},
+			},
+			{
+				Cachegroup: "org1",
+			},
+		},
+	}
+	_ = topology
```

Comment:
I think this line does literally nothing

Reply:
Removed do-nothing lines in 3b25d1aeae